src/Lykke.Job.BlockchainCashinDetector.Core/Domain/CashinAggregate.cs

```diff
@@ -24,6 +26,7 @@ public class CashinAggregate
         public string BlockchainAssetId { get; }
         public decimal Amount { get; }
         public string AssetId { get; }
+        public decimal OperationAmount { get; }
```

Comment:
As we have already discussed, let's rename Amount to the TransactionAmount, and TransactionAmount to the ActualTransactionAmount